shell/platform/common/accessibility_bridge_unittests.cc

```diff
@@ -337,5 +337,30 @@ TEST(AccessibilityBridgeTest, SliderHasSliderRole) {
   EXPECT_EQ(root_node->GetData().role, ax::mojom::Role::kSlider);
 }
 
+TEST(AccessibilityBridgeTest, canSetCheckboxChecked) {
```

Comment:
We've been so-so about this in the past, but we should get in the habit of adding a short comment above the test and if we're solving a particular edge-case or unobvious regression, consider adding an issue link. 

In this case, this is pretty basic functionality so an issue link is unnecessary, but potentially useful to add a comment.